Waiting for upstream resolution:

  90880 builds can fail due to non-portable glibc-specific internal macro usage
  https://bugs.freedesktop.org/show_bug.cgi?id=90880

--- pulseaudio-6.0/src/pulsecore/sample-util.h	2015-02-12 06:10:35.000000000 -0800
+++ pulseaudio-6.0/src/pulsecore/sample-util.h	2015-06-05 14:13:16.767036433 -0700
@@ -32,6 +32,15 @@
 #include <pulsecore/memblock.h>
 #include <pulsecore/memchunk.h>
 
+/* __WORDSIZE is a non-portable glibc-specific internal macro. */
+#if !defined(__WORDSIZE)
+# if defined(__LP64__)
+#  define __WORDSIZE	64
+# else
+#  define __WORDSIZE	32
+# endif
+#endif
+
 typedef struct pa_silence_cache {
     pa_memblock* blocks[PA_SAMPLE_MAX];
 } pa_silence_cache;
